tools/iop_dependencies.py

```diff
@@ -510,8 +510,13 @@ def add_edges(gr):
   gr.add_edge(('colorbalance', 'vibrance'))
```

Comment:
We probably want to delete this file since the module->priority has been removed from all iop, right?

Reply:
OK